app/components/contribution-list/template.hbs

```diff
@@ -1,41 +1,41 @@
-{{#if showQuickFilter}}
+{{#if @showQuickFilter}}
   <div class="quick-filter">
     <p>
       <label class="filter-contributor">
         Contributor:
-        <select onchange={{action (mut contributorId) value="target.value"}}>
+        <select onchange={{action (mut this.contributorId) value="target.value"}}>
           <option value="" selected>all</option>
-          {{#each contributorsActive as |contributor|}}
-            <option value={{contributor.id}} selected={{eq contributorId contributor.id}}>{{contributor.name}}</option>
+          {{#each this.contributorsActive as |contributor|}}
+            <option value={{contributor.id}} selected={{eq this.contributorId contributor.id}}>{{contributor.name}}</option>
           {{/each}}
         </select>
       </label>
 
       <label class="filter-contribution-kind">
         Kind:
-        <select onchange={{action (mut contributionKind) value="target.value"}}>
+        <select onchange={{action (mut this.contributionKind) value="target.value"}}>
           <option value="" selected>all</option>
-          {{#each contributionKinds as |kind|}}
-            <option value={{kind}} selected={{eq contributionKind kind}}>{{capitalize-string kind}}</option>
+          {{#each this.contributionKinds as |kind|}}
+            <option value={{kind}} selected={{eq this.contributionKind kind}}>{{capitalize-string kind}}</option>
           {{/each}}
         </select>
       </label>
 
       <label class="filter-contribution-size">
-        {{input type="checkbox" checked=hideSmallContributions}}
+        <Input @type="checkbox" @checked={{this.hideSmallContributions}} />
         Hide small contributions
       </label>
     </p>
   </div>
 {{/if}}
 
 <ul class="contribution-list">
-  {{#each contributionsFiltered as |contribution|}}
-    <li role="button" {{action "openContributionDetails" contribution}}
-        data-contribution-id={{contribution.id}}
-        class="{{contribution-status contribution}}{{if (eq contribution.id selectedContributionId) " selected"}}">
```

Comment:
Did you remove the use of the `contribution-status` helper because of the bug or is it just not needed here anymore?

Reply:
Yes, because of the bug. But this was an accidental commit. I just needed it locally to work on the rest.